dialogue-apache-hc5-client/src/main/java/com/palantir/dialogue/hc5/ApacheHttpClientBlockingChannel.java

```diff
@@ -88,12 +93,15 @@ public Response execute(Endpoint endpoint, Request request) throws IOException {
         } else if (requiresEmptyBody(endpoint)) {
             builder.setEntity(EmptyHttpEntity.INSTANCE);
         }
+        ClassicHttpRequest classicHttpRequest = builder.build();
+        long startTimeNanos = System.nanoTime();
         try {
-            CloseableHttpResponse httpClientResponse = client.apacheClient().execute(builder.build());
+            CloseableHttpResponse httpClientResponse = client.apacheClient().execute(classicHttpRequest);
```

Comment:
this includes handshake and idle connection validation time, so the p99s will be relatively large. I can look into starting the timer later in the request lifecycle.